### PR TITLE
remove lingering --

### PR DESCRIPTION
**Pull request by BenTheElder**
follow-up to https://github.com/kubernetes/test-infra/pull/28545/files#r1088270490

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml [hunk 1]
```diff
@@ -40,7 +40,6 @@ periodics:
       - runner.sh
       args:
       - make
-      - --
```
Comment (aojea) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml:43]: is the --directory a flag of `make`

Comment (BenTheElder) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml:43]: I don't think this was necessary before either, but it was dropped by `execute.py`.

None of the other jobs in the previous PR appear to have a bare `--` argument seperator.

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml:43]: yes, it's the long name for `-C`

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml:43]: https://www.gnu.org/software/make/manual/html_node/Options-Summary.html

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml:43]: execute.py only had a single flag, `--env`, which was not used by any of these jobs

**Comment by aojea** [comment]
TIL
/lgtm

**Comment by BenTheElder** [comment]
triggered rerun with latest config https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-build-and-push-k8s-at-golang-tip/1618704126743416832

**Pull request merged**